js/node/src/session_options_helper.cc

```diff
@@ -111,9 +126,11 @@ void ParseExecutionProviders(const Napi::Array epList, Ort::SessionOptions& sess
 #endif
 #ifdef USE_QNN
     } else if (name == "qnn") {
-      std::unordered_map<std::string, std::string> qnn_options;
-      qnn_options["backend_path"] = "QnnHtp.dll";
-      qnn_options["enable_htp_fp16_precision"] = "1";
+      // Ensure that the backend_path and enable_htp_fp16_precision options are set to default values if not provided.
```

Comment:
@fs-eire, @hectorSVC  if I'm understanding correctly, after https://github.com/microsoft/onnxruntime/pull/23486
was added, there's a mismatch between camel case and snake case keys expected by QNN EP?
WebGPU uses camel case for both ?

Reply:
looks like this is addressed in @fs-eire 's patch